extensions/wikia/BannerNotifications/css/BannerNotifications.monobook.scss

```diff
@@ -0,0 +1,6 @@
+@import 'skins/shared/styles/buttons/buttons-wikia-chiclet';
```

Comment:
It's really not a big deal, but I wish there were a more semantic place to put that include. 

Effectively this will include the chiclet styles in all of monobook, since the BannerNotifications extension is included on every page view. This is, of course, what we want, but what happens when we remove this extension and there were other places in monobook that were relying on that chiclet styling to be there? I wish we had a `monobook.scss` like we have `oasis.scss`, that included all the core css modules needed in monobook. Maybe monobook is just too far gone to be worrying about semantic code optimizations like this though. 

Just food for thought, not a blocker as far as I'm concerned.